karma.conf.js

```diff
@@ -27,13 +27,15 @@ module.exports = function(config) {
 
     files: [
       'tests/unit/**/*.js',
-      'static/js/modules/**/*.js'
+      'static/js/modules/**/*.js',
+      'static/js/vendor/**/*.js'
```

Comment:
I'm not a lawyer, but I think that unless someone rewrites tablist.js, it will remain a derivative work of the original file.